nav2_bringup/CMakeLists.txt

```diff
@@ -8,5 +8,8 @@ find_package(navigation2 REQUIRED)
 nav2_package()
 
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY map DESTINATION share/${PROJECT_NAME})
```

Comment:
Should 'map' be 'maps' and 'world' be 'worlds'? That's what we're using in the nav2_system_tests package.

Reply:
Yes, changed this.